crypto/bn/asm/x86_64-mont.pl

```diff
@@ -350,8 +368,15 @@
 .type	bn_mul4x_mont,\@function,6
 .align	16
 bn_mul4x_mont:
+.cfi_startproc
+___
+$code.=<<___ if ($addx);
+	mov	OPENSSL_ia32cap_P+8(%rip),%r11d
```

Comment:
This one is as well "run-across" thing. While it's totally correct that bn_mul4x_mont is never called, idea behind adding this was to improve clarity. I mean since bn_mul4x_mont is *declared* as function, it's probably appropriate if it looked as if it *could* be called directly, as actually worked if called. Maybe someone would call it in future... However! As I've spotted at least another similar spot, so I'll revert this modification and create separate request...